build.py

```diff
@@ -108,6 +108,8 @@ def md5(fname):
     'nvjpeg',
     'nvrtc',
     'nvrtc-builtins',
+    'nvJitLink',
+    'cupti'
```

Comment:
@danpetry this will avoid the need to bundle `cupti` separately like you did in https://github.com/AnacondaRecipes/cupti-feedstock. Unless you had a reason to do that?

Reply:
As far as I can recall, it was just because cupti wasn't part of cudatoolkit at that point. Can't see a strong reason why not.

Reply:
Apart from maybe ease of debugging (linker errors say cupti -> include cupti package)